analyzer/src/main/kotlin/managers/SpdxDocumentFile.kt

```diff
@@ -210,5 +211,16 @@ class SpdxDocumentFile(
     private fun getConcludedLicense(pkg: SpdxPackage): SpdxExpression? =
         pkg.licenseConcluded.takeIf { SpdxConstants.isPresent(it) }?.toSpdx()
 
-
+    /**
+     * Return a [RemoteArtifact] created with downloadLocation of the given package [SpdxPackage]
```

Comment:
I wonder how the scanner is supposed to use / download a `file://` URL.
Is it absolute, relative, relative to what?
Do we want to handle `file://` URLs in ORT?

@sschuberth @tsteenbe any thoughts?

Reply:
Using `file://` URIs here was actually something that I suggested in the chat as a work-around / first step towards supporting them in the downloader. I guess this simply is something we *need* to add support for in the context of project-style SPDX file that reference package directories within the same repository.

Reply:
My need is to be able to verify open source policy rules on C/C++ projects with a proprietary package manager. 
The problem is that the dependencies referenced in this package manager don't have VCS. They are only binaries (dll ...) or directories containing different files: .h, lib, dll ...
I quickly made a mapping downloadLocation - sourceArtifact
I'm not very satisfied with the use of the sourceArtifact in file:/ but it's a fast workaround at the moment and I'm trying to move forward like that.
Maybe a notion of folderArtifact on the ORT model side and/or a notion of dependency type (archive, folder) on the Spdx file side is missing?